src/client.ts

```diff
@@ -72,13 +69,13 @@ export class SpvWalletClient {
 
     if ('xPriv' in options) {
       this.logger.info('Using xPriv to sign requests');
-      this.xPriv = bsv.HDPrivateKey.fromString(options.xPriv);
+      this.xPriv = new HD().fromString(options.xPriv);
       return new HttpClient(this.logger, serverUrl, this.xPriv!, options.adminKey);
```

Comment:
I think that `!` is not needed anymore
```suggestion
      return new HttpClient(this.logger, serverUrl, this.xPriv, options.adminKey);
```

Reply:
Removed